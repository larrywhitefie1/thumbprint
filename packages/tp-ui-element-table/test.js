const sass = require('node-sass');

it('compiles correctly', () => {
    const css = sass
        .renderSync({ file: 'packages/tp-ui-element-table/_index.scss' })
        .css.toString();
    expect(css).toMatchSnapshot();
});
