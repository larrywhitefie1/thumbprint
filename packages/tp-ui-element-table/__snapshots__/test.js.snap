// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compiles correctly 1`] = `
".tp-table {
  width: 100%;
  font-size: inherit; }
  .tp-table th,
  .tp-table td {
    padding-left: 16px;
    padding-top: 8px;
    padding-bottom: 8px; }
    .tp-table th:first-child,
    .tp-table td:first-child {
      padding-left: 0; }
  .tp-table th {
    font-weight: 700;
    font-size: 20px;
    color: #676d73;
    text-align: left; }
    .tp-table th[field] {
      cursor: pointer; }
  .tp-table td {
    vertical-align: top; }
  .tp-table .tp-cell-collapse {
    width: 1%; }
  .tp-table--rule > thead > th, .tp-table--rule-bottom > thead > th {
    padding-bottom: 8px; }
  .tp-table--rule > thead > tr, .tp-table--rule-bottom > thead > tr {
    border-bottom: 1px solid #e9eced; }
  .tp-table--rule > tbody > tr, .tp-table--rule-bottom > tbody > tr {
    border-bottom: 1px solid #e9eced; }
    .tp-table--rule > tbody > tr:first-child, .tp-table--rule-bottom > tbody > tr:first-child {
      border-bottom: 1px solid #e9eced; }
    .tp-table--rule > tbody > tr:last-child, .tp-table--rule-bottom > tbody > tr:last-child {
      border-bottom: none; }
  .tp-table--rule-bottom > tbody > tr:last-child {
    border-bottom: 1px solid #e9eced; }
"
`;
